diff --git a/node_modules/react-native-google-mobile-ads/src/specs/modules/NativeAppModule.ts b/node_modules/react-native-google-mobile-ads/src/specs/modules/NativeAppModule.ts
--- a/node_modules/react-native-google-mobile-ads/src/specs/modules/NativeAppModule.ts
+++ b/node_modules/react-native-google-mobile-ads/src/specs/modules/NativeAppModule.ts
@@ -16,24 +16,18 @@
 
 import type { TurboModule } from 'react-native';
 import { TurboModuleRegistry } from 'react-native';
-import type { CodegenTypes } from 'react-native';
+import type { UnsafeObject } from 'react-native/Libraries/Types/CodegenTypes';
 
 export interface Spec extends TurboModule {
   // App initialization
-  initializeApp(
-    options: CodegenTypes.UnsafeObject,
-    appConfig: CodegenTypes.UnsafeObject,
-  ): Promise<CodegenTypes.UnsafeObject>;
+  initializeApp(options: UnsafeObject, appConfig: UnsafeObject): Promise<UnsafeObject>;
   setAutomaticDataCollectionEnabled(appName: string, enabled: boolean): void;
   deleteApp(appName: string): Promise<void>;
 
   // Events
   eventsNotifyReady(ready: boolean): void;
-  eventsGetListeners(): Promise<CodegenTypes.UnsafeObject>;
-  eventsPing(
-    eventName: string,
-    eventBody: CodegenTypes.UnsafeObject,
-  ): Promise<CodegenTypes.UnsafeObject>;
+  eventsGetListeners(): Promise<UnsafeObject>;
+  eventsPing(eventName: string, eventBody: UnsafeObject): Promise<UnsafeObject>;
   eventsAddListener(eventName: string): void;
   eventsRemoveListener(eventName: string, all: boolean): void;
 
@@ -42,15 +36,15 @@
   removeListeners(count: number): void;
 
   // Meta
-  metaGetAll(): Promise<CodegenTypes.UnsafeObject>;
+  metaGetAll(): Promise<UnsafeObject>;
 
   // JSON
-  jsonGetAll(): Promise<CodegenTypes.UnsafeObject>;
+  jsonGetAll(): Promise<UnsafeObject>;
 
   // Preferences
   preferencesSetBool(key: string, value: boolean): Promise<void>;
   preferencesSetString(key: string, value: string): Promise<void>;
-  preferencesGetAll(): Promise<CodegenTypes.UnsafeObject>;
+  preferencesGetAll(): Promise<UnsafeObject>;
   preferencesClearAll(): Promise<void>;
 }
 
